test/unit/DolaFlashMinter.test.ts

```diff
@@ -0,0 +1,196 @@
+import { expect } from "chai";
+import { MockContract } from "ethereum-waffle";
+import { BigNumber, constants, Contract, utils } from "ethers";
+import { waffle } from "hardhat";
+import IERC3156FlashBorrowerABI from "../../artifacts/contracts/interfaces/IERC3156FlashBorrower.sol/IERC3156FlashBorrower.json";
+import MockDolaABI from "../../artifacts/contracts/test/MockDola.sol/MockDola.json";
```

Comment:
`MockDola.sol` looks like a file, not a folder

Reply:
That's the compiler output format